json_object.h

```diff
@@ -899,6 +899,28 @@ extern int json_object_set_string_len(json_object* obj, const char* new_value, i
 extern int json_object_equal(struct json_object *obj1,
 			     struct json_object *obj2);
 
+/**
+ * Copy the contents of the JSON object.
+ * If the destination object is NULL, it will be allocated.
+ * If the destination object is non-NULL, it will be overwrriten
```

Comment:
before addressing this, let's resolve this comment first :)
https://github.com/json-c/json-c/pull/328/commits/9f420cdf94b1c83800cae66b22f035f8182ac5c3#r126346219